astrology.lic

```diff
@@ -265,7 +267,7 @@ class Astrology
     check_ripples
     check_astral
     check_weather
-    check_events(check_pools)
+    check_events(check_pools) if @predict_event
```

Comment:
I considered doing the check as a guard in `check_events` but i wanted to leave open the possibility that you could add a command arg to the script for predicting events and be independent of this training setting.